src/test/test-fs-util.c

```diff
@@ -850,11 +850,12 @@ static void test_path_is_encrypted_one(const char *p, int expect) {
         int r;
 
         r = path_is_encrypted(p);
-        if (r == -ENOENT) /* This might fail, if btrfs is used and we run in a container. In that case we
-                           * cannot resolve the device node paths that BTRFS_IOC_DEV_INFO returns, because
-                           * the device nodes are unlikely to exist in the container. But if we can't stat()
-                           * them we cannot determine the dev_t of them, and thus cannot figure out if they
-                           * are enrypted. Hence let's just ignore ENOENT here. */
+        if (IN_SET(r, -ENOENT, -EACCES)) /* This might fail, if btrfs is used and we run in a container. In
```

Comment:
use ERRNO_IS_PRIVILEGE() since different container mgrs might protect this differently